app/controllers/ships_controller.rb

```diff
@@ -1,17 +1,13 @@
 class ShipsController < ApplicationController
 
   def index
-  end
-
-  def create
-    @ship = Ship.create(ship_params)
-    render action: "index"
+    @ships = Ship.where("game_id = ? AND user_id = ?", params[:game_id], :user_id)
```

Comment:
I don't think the `:user_id` symbol here is doing what you think.  I think you should instead have:

`current_user.id`

Are the @ships actually used in the create action?